.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,50 @@
+#
+name: Create and publish a Docker image
+
+# Configures this workflow to run every time a change is pushed to the branch called `release`.
+on:
+  push:
+    branches:
+          -  containers
```

Comment:
The branch should be main, to prevent pushing images under development to the containers registry. Maybe we could have a workflow for main and dev branches ("-dev" in the image name should be self-explanatory)

Reply:
removed GH actions, since it is not used currently